packages/ui/src/TextArea.module.scss

```diff
@@ -0,0 +1,116 @@
+@use '../styles/constants' as c;
+@use '../styles/helpers' as h;
+
+.container {
+  @include c.typographyBodySmall;
+
+  color: c.$colorText;
+  box-sizing: border-box;
+  margin: 0;
+
+  .label {
+    margin-bottom: c.$grid;
+  }
+
+  .helperText {
+    display: block;
+    margin: c.$grid 0 0 c.$grid;
+  }
+
+  .errorContainer {
+    margin-top: c.$grid;
+  }
+
+  .textAreaContainer {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .textAreaWrapper {
+    position: relative;
+    width: 100%;
+  }
+
+  .focusOverlay {
+    position: absolute;
+    display: block;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    pointer-events: none;
+    z-index: c.$zIndex100;
+    border-radius: c.$borderRadius;
+    visibility: hidden;
+
+    @include h.outlineSimple;
+  }
+
+  textarea {
+    display: block;
+    padding: c.$grid * 2 c.$grid * 3;
+    border: c.$borderWidth solid c.$colorNeutralLight;
+    box-sizing: border-box;
+    width: 100%;
+    min-width: 100%;
+    font-family: inherit;
+    font-weight: inherit;
+    font-size: inherit;
+    line-height: inherit;
+    letter-spacing: inherit;
+    color: inherit;
+    border-radius: c.$borderRadius;
+    background-color: c.$colorNeutralLighter;
+    /* doiuse-disable css-resize */
+    resize: vertical;
+    /* doiuse-enable css-resize */
+
+    // Note: Resize is not supported by IE11, it's presence does no harm though
+    &.notResizable {
+      /* doiuse-disable css-resize */
+      resize: none;
+      /* doiuse-enable css-resize */
+    }
+
+    &::placeholder {
+      color: c.$colorTextSecondary;
+    }
+  }
+
+  &.empty {
+    color: c.$colorTextSecondary;
+  }
```

Comment:
I didn't notice it in `TextField` but why do we need it? Isn't `placeholder` selector enough?

Reply:
For the TextField we need to color the icon inside of it. Here we do not need it indeed. Nice catch!